### ISSUE TITLE
A11y_SQLdatabase projects extension_View And More Action_Update Project From Database_Name Role Value: Name is not descriptive for the radio buttons which are present in the "Action" section.

### ISSUE THREAD

**Issue by SachinChavhan317** [microsoft/azuredatastudio#22133]
### GitHub Tags:
#A11yWCAG2.1; #A11ySev3; #A11yTCS; #DesktopApp; #SH-SQL Azure Data Studio-MAC-March23; #SQL Azure Data Studio; #Name Role Value; #WCAG4.1.2; #MAC; #AINotInScope; #Voiceover; 

### Environment Details
Application Name: SQL Azure Data Studio
Version: 1.42.0-insider (Universal)
MAC Version: macOS Ventura 13.3

### Repro Steps

1. Start Voiceover with using shortcut key Command+F5.
2. Launch Azure Data Studio
3. Tab till extension and search for “SQL Database Projects” and install it.
4. Tab till database project and press enter. “Database Project” panel will open.
5. Tab till “View and More Action” button and press enter.
6. Tab till “Update Project from Database” and press enter.
7. Tab till radio buttons which are present in the "Action" section.
8. Verify whether the name is descriptive for the radio buttons which are present in the "Action" section or not.

### Actual Result:​
Name is not descriptive for the radio buttons which are present in the "Action" section.

**Note issue:**
Publish Target: Voiceover is not announcing publish target radio group name.

### Expected Result:​
Name should be more descriptive for the radio buttons which are present in the "Action" section.
Voiceover must read the name of the radio button along with the section name as action.

### User Impact:
Voiceover user will not be able to get the proper information about the radio buttons.

### Attachment:
![Presentation1](https://user-images.githubusercontent.com/86241494/222706464-74b4502d-8b9d-4b9b-8c04-df3da1bf5e44.gif)
**Note issue:**
![publish - label for publish target](https://user-images.githubusercontent.com/86241494/223635527-42833807-7788-49b6-b057-8829f919e378.gif)

**Comment by kisantia** [comment]
@SachinChavhan317 this has been fixed in the latest version of the sql database projects extension (0.23.0). Please download the [latest insiders build](https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch) and verify this is fixed. Thanks!

**Comment by kisantia** [comment]
@SachinChavhan317 Please verify this issue is fixed.

**Comment by SachinChavhan317** [comment]
Verified the bug on version: 1.44.0-insider (Universal).
The issue is fixed, hence closing the bug. #Closed;

Please refer below attachment for the reference.
![22446_Action](https://user-images.githubusercontent.com/86241494/233293400-3e0be736-5982-470c-b51e-5af2a26a8380.gif)
![22446_Publish](https://user-images.githubusercontent.com/86241494/233293422-cd98a106-2293-4c40-bef1-9d61c002a8c2.gif)
